packages/web-api/src/types/response/ReactionsGetResponse.ts

```diff
@@ -21,7 +20,8 @@ export type ReactionsGetResponse = WebAPICallResult & {
 
 export interface Message {
   app_id?: string;
-  blocks?: MessageBlock[];
+  assistant_app_thread?: AssistantAppThread;
+  blocks?: AssistantAppThreadBlock[];
```

Comment:
This can't be correct, can it? Also, `MessageBlock` below being replaced by an Assistant-specific type. @filmaj @seratch

Reply:
It's just the type generation's obtuse naming. If you look at what the definition of these things are, they're the same as before, but for whatever reason the type generator decided to name the interface after assistant stuff 🤷

Reply:
The naming is done by quicktype and it's not controllable. If a 3rd party developer has the exported name in their code, indeed this could be a change. However, most developers won't care what the name of these internal types. Also, there is no way for us to rever this.

Reply:
Alright, it sounds like we're OK with this change then. Thanks for confirming 🙇🏼‍♀️